### PR TITLE
GUVNOR-3486: [Guided Decision Table Graph] Refresh after removal of table

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-3486

(Actually see [here](https://issues.jboss.org/browse/RHBRMS-2955?focusedCommentId=13504046&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13504046)).

### CODE REVIEW

**Comment by manstis** [comment]
Part of an ensemble:
- https://github.com/kiegroup/kie-wb-common/pull/1316
- https://github.com/kiegroup/drools-wb/pull/726

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/BaseGuidedDecisionTableEditorPresenter.java [hunk 1]
```diff
@@ -266,6 +266,7 @@ protected void onDecisionTableSelected(final DecisionTableSelectedEvent event) {
                        MenuItems.VALIDATE);
 
         if (!dtPresenter.isPresent()) {
+            activeDocument = null;
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/BaseGuidedDecisionTableEditorPresenter.java]: This is the cause of the problem reported in @jomarko comment on the JIRA.

**Comment by manstis** [comment]
@jomarko I know you're winding down for Christmas (or, more likely, rushed off your feet).. but it'd be nice to get this merged before Christmas...

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Pull request merged**